src/schema/v2/previewSavedSearch.ts

```diff
@@ -137,30 +153,65 @@ const getRaritySearchCriteriaLabel = (
     }
   }
   return {
-    displayValue: attributionClasses[name].name,
+    displayValue: attributionClasses[value].name,
     field: "attributionClass",
-    value: name,
+    value,
     name: "Rarity",
   }
 }
 
+const getMediumSearchCriteriaLabel = (
+  value: string | undefined
+): SearchCriteriaLabel | null => {
+  if (!value) {
+    return null
+  }
+
+  const mediumData = Object.values(artworkMediums).find((artworkMedium) => {
+    return artworkMedium.mediumFilterGeneSlug === value
+  })
+
+  if (!mediumData || !mediumData.mediumFilterGeneSlug) {
+    return null
+  }
+
+  return {
+    displayValue: mediumData.name,
+    field: "additionalGeneIDs",
```

Comment:
we call medium `additionalGeneIdDs` in our client apps therefore the naming here